pkg/models/dashboards.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
+const RootFolderName = "General"
```

Comment:
I think this makes sense, because it's better to use `const` when a variable is actually a constant.